src/quo/components/text_combinations/view.cljs

```diff
@@ -1,5 +1,6 @@
 (ns quo.components.text-combinations.view
```

Comment:
this component namespace is wrong, do you know which component is this supposed to be?
<img width="236" alt="Screenshot 2023-10-31 at 11 43 42" src="https://github.com/status-im/status-mobile/assets/22799766/cf440b51-d363-49d6-a260-d9194e12b823">
seems like it should be `standard title`?

Reply:
Yes the component needs some refactoring. There is an open issue https://github.com/status-im/status-mobile/issues/17456

Reply:
ok thanks Omar 👍